ibm/service/power/resource_ibm_pi_shared_processor_pool_test.go

```diff
@@ -0,0 +1,144 @@
+// Copyright IBM Corp. 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"testing"
+
+	acc "github.com/IBM-Cloud/terraform-provider-ibm/ibm/acctest"
+
+	"github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccIBMPISPPbasic(t *testing.T) {
+	name := fmt.Sprintf("tfpispp%d", acctest.RandIntRange(10, 100))
```

Comment:
```suggestion
	name := fmt.Sprintf("tf-pi-spp-%d", acctest.RandIntRange(10, 100))
```
For easy of readability

Reply:
I cannot add hyphens, but I can add underscores.